class/student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
- [ ]  Kindly make sure to call `super` with Required Arguments. In the `Student` class constructor, you're calling `super` without passing the required arguments that the parent class `Person` expects. Make sure to pass the `age`, `parent_permisson`, and `name` arguments to the `super` call in the `initialize` method of the `Student` class, so the parent class can initialize properly.